channel-sdk/src/main/java/com/zello/channel/sdk/ChannelFeature.kt

```diff
@@ -0,0 +1,8 @@
+package com.zello.channel.sdk
+
+// TODO: Document ChannelFeature
```

Comment:
Are you planning on a separate pass at adding the missing documentation later?

Reply:
Yes, after all features are implemented I'm going to make a pass to make sure everything is documented. I can document this now though if you'd be more comfortable that way.